app/embeds/GoogleDrawings.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import GoogleDrawings from "./GoogleDrawings";
+
+describe("GoogleDrawings", () => {
+  const match = GoogleDrawings.ENABLED[0];
+  test("to be enabled on share link", () => {
+    expect(
+      "https://docs.google.com/drawings/d/1zDLtJ4HSCnjGCGSoCgqGe3F8p6o7R8Vjk8MDR6dKf-U/edit".match(
```

Comment:
I think the `/edit` link is not the shared link. From my experience, it is with ending `?usp=sharing` only.

Reply:
I dont thin this is correct.

The following links is provided by my gdrive app when creating a link. Even if the link provides read only access.

```
https://docs.google.com/drawings/d/1zDLtJ4HSCnjGCGSoCgqGe3F8p6o7R8Vjk8MDR6dKf-U/edit?usp=drivesdk
```

Reply:
Then please put it also into your tests.

Reply:
it is already part of the tests.

see https://github.com/outline/outline/pull/1814/files#diff-dad70d662be2d7eda7c97042f853eff786b32a266e7a93f8581bc032d8ee575cR13

i explicit want to allow links without the parameter to be able to integrate drawings that are not published.